docs/UserGuide.md

```diff
@@ -3,7 +3,7 @@ layout: page
 title: User Guide
 ---
 
-ClinicConnectSystem Level 3 (AB3) is a **desktop app for managing contacts, optimized for use via a Command Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, AB3 can get your contact management tasks done faster than traditional GUI apps.
+ClinicConnectSystem is a **desktop app for managing contacts, optimized for use via a Command Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, AB3 can get your contact management tasks done faster than traditional GUI apps.
```

Comment:
I think our product can be called "ClinicConnect" instead of "ClinicConnectSystem"? But we don't need to change the name of our class cos the code is basically ClinicConnect's system. What do you think?